client/src/components/Note.tsx

```diff
@@ -0,0 +1,16 @@
+import { MdDeleteForever } from "react-icons/md";
+
+const Note = () => {
+  var date = new Date().getDate();
```

Comment:
```suggestion
  var date = new Date();
```

`new Date().getDate()` will return day of the month only.